Composer/packages/client/src/pages/design/DebugPanel/TabExtensions/DiagnosticsTab/DiagnosticType.ts

```diff
@@ -196,6 +263,10 @@ export class LuDiagnostic extends DiagnosticInfo {
   ) {
     super(rootProjectId, projectId, id, location, diagnostic);
     this.dialogPath = this.findDialogPath(luFile, dialogs, diagnostic);
+    const friendlyPath = getFriendlyPath(this.dialogPath, dialogs);
+    if (friendlyPath.length) {
```

Comment:
does it matter if it's set to an empty [] vs. being undefined

Reply:
In both the cases we would just display the location property. (non friendly one)